### PR TITLE
MC-14261 Added doc for get firewall rule (list and single)

### PR DESCRIPTION
**Pull request by aaronalmeida**
#### Changes made

- Added API docs for listing all firewall rules and for a single rule 

#### Related PRs
- [MC-13925](https://cloudops.atlassian.net/browse/MC-13925) (Parent Story, listing all rules)
- [MC-14262](https://cloudops.atlassian.net/browse/MC-14262) (Story where we use the GET for a single rule)

### CODE REVIEW

File: source/includes/stackpath/_firewallrules.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    "enabled": "true"
+    },
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrules/:id?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Query Params | &nbsp;
+---- | -----------
+`siteId`<br/>*UUID* | The ID of the site for which the firewall rule resides in. This parameter is required.
```
Comment (razine-bensari) [inline source/includes/stackpath/_firewallrules.md]: I think the following is more accurate. Let me know what you think.
`The ID of the site for which the firewall rule will be applied to`

Comment (razine-bensari) [reply source/includes/stackpath/_firewallrules.md]: Same for above if you decide to change it

**Review by razine-bensari — APPROVED** [review]
Noice!

File: source/includes/stackpath/_firewallrules.md [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+  ],
+  "metadata": {
+    "recordCount": 2
+  }
+}
+```
+
+<code>GET /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/firewallrules?siteId=<a href="#stackpath-sites">:siteId</a></code>
+
+Query Params | &nbsp;
```
Comment (monil-panchal) [inline source/includes/stackpath/_firewallrules.md:44]: @aaronalmeida 
This requires adding one more optional param - `action` which can filter the rules based on `ALLOW/BLOCK`

I'll do this change.

Comment (aaronalmeida) [reply source/includes/stackpath/_firewallrules.md:44]: @monil-panchal Sounds good!

File: source/includes/stackpath/_firewallrules.md (hunk 1 above)
Comment (aaronalmeida) [reply source/includes/stackpath/_firewallrules.md]: ec292889f28eaa95398c4d14fa44d3683e19e706

File: source/includes/stackpath/_firewallrules.md (hunk 2 above)
Comment (monil-panchal) [reply source/includes/stackpath/_firewallrules.md:44]: Added here - https://github.com/cloudops/cloudmc-api-docs/pull/339/commits/aea0eea9fca6ee06ae04b292b9490963f626be70

@aaronalmeida @razine-bensari Is it looking good or needs some changes?

**Review by monil-panchal — APPROVED** [review]

File: source/includes/stackpath/_firewallrules.md (hunk 2 above)
Comment (aaronalmeida) [reply source/includes/stackpath/_firewallrules.md:44]: @monil-panchal Thanks, looks good to me!

**Pull request merged**